ballerina-test-automation/gradlew

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env sh
```

Comment:
@NipunaMadhushan Can we remove this Gradle wrapper as there is one in the repo root?

Reply:
@keizer619 Removed the gradle wrapper in sub module "ballerina-test-automation". Now it uses the gradle wrapper in repo root.